source/render/ODFrameListener.cpp

```diff
@@ -92,8 +92,6 @@ ODFrameListener::ODFrameListener(Ogre::RenderWindow* renderWindow, Ogre::Overlay
 
     mGameMap = new GameMap(false);
 
-    mMiniMap = new MiniMap(mGameMap);
```

Comment:
I must say at first I wondered why you duplicated the minimap in each mode. But after thinking about it, it makes sense, for not all the modes will use a minimap.